src/app/admin/page.tsx

```
@@ -0,0 +1,28 @@
+import LoginButton from "@/components/LoginButton";
+
+import LoginButton from "@/components/LoginButton";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'LoginButton'. Consider to delete it or rename it.

'LoginButton' is defined here:


(lint/suspicious/noRedeclare)